apps/gallery/style/info.css

```diff
@@ -40,6 +40,14 @@
   border-bottom: none;
 }
 
+.no-border {
+ border-bottom: none !important;
+}
+
+.hide {
+  display: none !important;
+}
+
```

Comment:
This is not a scoped stylesheet, so what you add here applies to the whole app.  I think that these classes should be prefixed with `#info-view > section > dl > dd ` or something like that.

And if you do that, then you might not need the !important and the xfail.list changes.

Alternatively you could avoid the stylesheet changes altogether by explicitly setting inline styles (style.display = 'none';) in the JS file.  That would simplify the patch to just a single file.